### ISSUE TITLE
Agreement with sixtrack degraded between sixtrack v5.1.3 and v5.2

### ISSUE THREAD

**Issue by giadarol** [rdemaria/sixtracklib#25]
To test:
```
cd examples/python/test_footprint_from_sixtrack_input
python 000_pickle_line_and_closed_orbit.py
python 001_checks_against_sixtrack.py
python 002_compare_individual_particle.py
```
with sixtrack 5.1.3:
![v5 1 3](https://user-images.githubusercontent.com/8049893/60824199-78382080-a1a9-11e9-97d5-f4e978980e9c.png)

with sixtrack 5.2:
![v5 2](https://user-images.githubusercontent.com/8049893/60824189-71111280-a1a9-11e9-81e2-9dd7e308bd58.png)

**Comment by rdemaria** [comment]
I cannot reproduce the last results, I always get very similar errors in the order of 1e-10 with different versions.

**Comment by giadarol** [comment]
Are you rerunning all three scripts? The error might be in the computation of the closed orbit, that I then use to initialize the sixtracklib machine

**Comment by rdemaria** [comment]
Yes, the 3 scripts...

**Comment by giadarol** [comment]
Let's have a look together...

**Comment by rdemaria** [comment]
A possible explanation is that
* 5.1 uses delta in fort.13 to recompute energy
* 5.2.x uses energy in fort.13 to recompute delta

The example does not use enough digits and the difference is more visible:

5.1
```
     ---- INITIAL COORD. OF TWIN-TRAJECTORIES
                      0.098238169999999999659756610981276
                      0.000977765600000000002084421524273
                      0.199936100000000005483258291860693
                      0.002999126999999999869384037509690
                      2.004004000000000118575371743645519
                      0.000003420192000000000084127026526
                      0.098238169999999999659756610981276
                      0.000977765600000000002084421524273
                      0.199936100000000005483258291860693
                      0.002999126999999999869384037509690
                      2.004004000000000118575371743645519
                      0.000003420192000000000084127026526
                7000000.000000000000000000000000000000000
                7000023.941343570128083229064941406250000
                7000023.941343570128083229064941406250000
``` 

5.2
```
     ---- INITIAL COORD. OF TWIN-TRAJECTORIES
                      0.098238169999999999659756610981276
                      0.000977765600000000002084421524273
                      0.199936100000000005483258291860693
                      0.002999126999999999869384037509690
                      2.004004000000000118575371743645519
                      0.000003428571490237692487014443607
                      0.098238169999999999659756610981276
                      0.000977765600000000002084421524273
                      0.199936100000000005483258291860693
                      0.002999126999999999869384037509690
                      2.004004000000000118575371743645519
                      0.000003428571490237692487014443607
                7000000.000000000000000000000000000000000
                7000024.000000000000000000000000000000000
                7000024.000000000000000000000000000000000
```

**Comment by giadarol** [comment]
The reference energy was not properly initialized.
 - Fixed in  #26
